README.md

```diff
@@ -116,6 +116,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Wheel Ride](http://drawvr.com/wheel/) (by [@drawvr])
 - [Equalizer Ride](http://drawvr.com/equalizer/) (by [@drawvr])
 - [Test Scape](http://x627.com/testscape/) (by [Michael Verdi](http://x627.com/directory/))
+- [INFIVERSE](http://infiverse.com/) (by [@drawvr])
```

Comment:
Infiverse= Collective of responsive 3D & WebVR websites made in MozVR A-Frame. Find cool experiences, templates, code snippets and more